boot/zephyr/nrf_cleanup.c

```diff
@@ -92,6 +93,19 @@ void nrf_cleanup_peripheral(void)
         nrfy_uarte_event_clear(current, NRF_UARTE_EVENT_RXTO);
         nrfy_uarte_disable(current);
 
+        uint32_t pin[4];
+
+        pin[0] = nrf_uarte_tx_pin_get(current);
+        pin[1] = nrf_uarte_rx_pin_get(current);
+        pin[2] = nrf_uarte_rts_pin_get(current);
+        pin[3] = nrf_uarte_cts_pin_get(current);
```

Comment:
this mixes `nrfy_uarte` and `nrf_uarte`, should probably stick to `nrfy_uarte` and use `nrfy_uarte_pins_get` or just use `nrfy_uarte_pins_disconnect`